drivers/can/Kconfig.mcp2515

```diff
@@ -0,0 +1,80 @@
+# Kconfig.mcp2515 - MCP2515 CAN configuration options
+
+#
+# Copyright (c) 2018 Karsten Koenig
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config CAN_MCP2515
+	bool "MCP2515 CAN Driver"
+	depends on SPI
+	help
+	  Enable MCP2515 CAN Driver
+
+if CAN_MCP2515
+
+config CAN_MCP2515_OSC_FREQ
```

Comment:
I guess these could/should go into device tree as well?
- CAN_MCP2515_OSC_FREQ
- CAN_PROP_SEG
- CAN_PHASE_SEG1
- CAN_PHASE_SEG2
- CAN_SJW

Reply:
Hm yes, the segment length ones are sort of defined already in the can devicetree, but seg1 and prop are merged together there but the mcp2515 allows specifing them independently. I'll try sorting out again why I didn't just use it before

Reply:
@karstenkoenig it should be save to merge CAN_PROP_SEG and CAN_PHASE_SEG1. Even the CAN standard says so. I have no idea why this is separate in the mcp2515. They have the same effect.